.github/workflows/continuous-integration.yml

```diff
@@ -17,16 +17,14 @@ jobs:
           - macos-latest
         rust_channel:
           - stable
-          - beta
-          - nightly
         include:
           - os: ubuntu-latest
             system: x86_64-linux
             experimental: false
           - os: macos-latest
             system: x86_64-darwin
             experimental: false
-          - rust_channel: nightly
+          - rust_channel: stable
```

Comment:
This is the qualifier for `experimental: true`, which specifies that the build can fail on nightly but that shouldn't abort the whole CI, because nightly is more subject to break. We don't want `stable` to be experimental, so we probably rather want to delete this line and the next one. We may keep the other references to `experimental` even if this is now useless for now I guess, as those may still serve if we want to add experimental tests at some point.